client/js/components/comments/comment-list.jsx

```diff
@@ -34,7 +34,7 @@ export default class CommentList extends React.Component {
   }
 
   render() {
-    let comments = this.props.comments.map((comment) => {
+    let comments = this.props.comments && this.props.comments.map((comment) => {
```

Comment:
@blimusiek this is another kind of if-fix , should i delete it and let the comments brake on snippet deleteing?